dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -77,6 +99,14 @@ function get_disk_list {
             continue
         fi
         disk_size=$(echo "${disk_meta}" | cut -f2 -d:)
+        if [ ${max_disk} -gt 0 ]; then
+            local disk_size_b
```

Comment:
I usually name variables that take byte sizes such that the unit is clear, disk_size_b was clear to me at assignment time but before I thought if there is disk_size_b where is disk_size_a :) Could we make this clear too. e.g

```
disk_size_bytes
```

This naming would then be consistent with other parts of code I wrote e.g in kiwi-partitions-lib.sh
Thanks